modules/installation-preparing-restricted-cluster-to-gather-support-data.adoc

```diff
@@ -9,9 +9,15 @@ Clusters using a restricted network must import the default must-gather image in
 
 .Procedure
 
-. Import the default must-gather image from your installation payload:
+* Import the default must-gather image from your installation payload:
 +
 [source,terminal]
 ----
 $ oc import-image is/must-gather -n openshift
 ----
+** If you are in an isolated network or disconnected setting, use the `--image` flag as part of must-gather and point to the payload image, as in the following example:
```

Comment:
as part of must-gather -> with the `oc adm must-gather` command

Reply:
Also, this module is specifically for users in a restricted network, so I don't think that "If you are in an isolated network or disconnected setting" is appropriate.

I would also consider not making this a subbullet. And instead would just bump it down as a sentence after the procedure. Like:

```
.Procedure

* Import the default must-gather image from your installation payload:
+
[source,terminal]
----
$ oc import-image is/must-gather -n openshift
----

When running the `oc adm must-gather` command, use the `--image` flag and point to the payload image, as in the following example:

[source,terminal]
----
$ oc adm must-gather --image=$(oc adm release info --image-for must-gather)
----
```